staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -61,6 +62,9 @@ const (
 	defaultBatchBufferSize = 1000        // Buffer up to 1000 events before blocking.
 	defaultBatchMaxSize    = 100         // Only send 100 events at a time.
 	defaultBatchMaxWait    = time.Minute // Send events at least once a minute.
+
+	defaultBatchThrottleQPS   = 10 // Send routine at 10 qps.
+	defaultBatchThrottleBurst = 15 // Send routine burst is 15.
```

Comment:
How did you come up with these numbers?

IIUC, this is basically limiting the steady-state QPS of audited calls to 1000? That strikes me as low.

Reply:
I planned to change them after this PR, with introducing the possibility to configure them. But setting them now also seems like a good idea

From my observations, 50 QPS in a steady state should be enough for a current scalability goals (1000 events per second is the current steady state of the 1k cluster under performance tests, and that's in version 1.7, with RequestReceived stage)

Reply:
Yes, normally 10 routines/s at burst 15. I think it's enough. But better to expose to us to configure.

Reply:
Maybe 50 is appropriate, I mistake  100 as 1000. I will change it to 50. Thank you all.